scripts/spcgeonode/geoserver/docker-entrypoint.sh

```diff
@@ -100,15 +104,17 @@ fi
 sed -i -r "s|<cliendId>.*</cliendId>|<cliendId>$CLIENT_ID</cliendId>|" "/spcgeonode-geodatadir/security/filter/geonode-oauth2/config.xml"
 sed -i -r "s|<clientSecret>.*</clientSecret>|<clientSecret>$CLIENT_SECRET</clientSecret>|" "/spcgeonode-geodatadir/security/filter/geonode-oauth2/config.xml"
 # OAuth endpoints (client)
-sed -i -r "s|<userAuthorizationUri>.*</userAuthorizationUri>|<userAuthorizationUri>/o/authorize/</userAuthorizationUri>|" "/spcgeonode-geodatadir/security/filter/geonode-oauth2/config.xml"
+# These must be reachable by user
+sed -i -r "s|<userAuthorizationUri>.*</userAuthorizationUri>|<userAuthorizationUri>$GEONODE_URL/o/authorize/</userAuthorizationUri>|" "/spcgeonode-geodatadir/security/filter/geonode-oauth2/config.xml"
```

Comment:
Can someone review this line?
I was a little bit confused that the original entry didn't contain the base URL.
It works because it was set directly in the file and use absolute URI.
However if we open OAuth2-geonode settings page in GeoServer, we will not be able to save it again because it didn't pass frontend validations.
I propose to use the full URL here.